test/e2e/framework/nodes_util.go

```diff
@@ -75,8 +75,8 @@ func AppendContainerCommandGroupIfNeeded(args []string) []string {
 	return args
 }
 
-// MasterUpgradeGKE upgrades master node to the specified version on GKE.
-func MasterUpgradeGKE(namespace string, v string) error {
+// MasterControlPlaneUpgradeGKE upgrades master node to the specified version on GKE.
+func MasterControlPlaneUpgradeGKE(namespace string, v string) error {
```

Comment:
Need to remove `Master` from the function name.

Reply:
fine